src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/ConfirmationStatementSubmission.java

```diff
@@ -12,14 +12,18 @@
     @Id
     private String id;
 
+    @Field("data")
+    private String data;
```

Comment:
It would be more extendable going forward if data was it's own Object.
Something like ConfirmationStatementSubmissionData.
Which contains it's own fields.
Such as StatementOfCapitalData

Reply:
Was unsure whether this should be a string, a map or an object. What do you envision this class to look like in terms of types and fields?

Reply:
Was unsure whether this should be a string, a map or an object. There did not seem to any specifications here, so we should be clear. What do you envision this class to look like in terms of types and fields?